src/components/track/desktop/ConnectedTrackTile.tsx

```diff
@@ -166,7 +166,7 @@ const ConnectedTrackTile = memo(
         includeShareToTikTok: !is_unlisted,
         includeTrackPage: true,
         isArtistPick: isArtistPick,
-        isDeleted: is_delete,
+        isDeleted: is_delete || !!user?.is_deactivated,
```

Comment:
is the `!!` needed? `is_deactivated` should be a boolean right?

Reply:
Yeah that's correct, I should be able to remove these now. Was thinking it could be null/undefined

Reply:
Right, the `user` might be null/undefined (or is it possible?), so that's why I double banged

Reply:
in practice we should always have the user here. reason we have this is once upon a time we rendered the track tile w/o having the full user (yet). we should clean up these types, but this change seems ~reasonable for now.

FWIW, we have this weird fallback code above https://github.com/AudiusProject/audius-client/pull/657/commits/89ef4e1029082d7675a46f28f6be4efcbc08985c#diff-05e159c9c0c0c18c159b331ace712a645a34ba2a3b9f41357a25e267c82f44daR114